duplas/Dupla_André_Julio.md

```diff
@@ -0,0 +1,19 @@
+# Dupla ANDRÉ e JULIO
```

Comment:
Falta ainda alterar o arquivo duplas.md com o link para o arquivo criado.

Reply:
Feito

Reply:
Criei um pull resquest chamado “Atualização de dupla”. Com a alteração do duplas.md

Reply:
OK. De maneira geral, quando alterações forem pedidas, essas precisam ser feitas no mesmo PR. Se for feito em outro PR, isso cria margem para inconsistencia de dados. Por hora vou aceitar os dois, mas posteriormente mande somente um.